content/en/contribute/development.md

```diff
@@ -154,7 +154,7 @@ go install
 Hugo relies on [mage](https://github.com/magefile/mage) for some convenient build and test targets. If you don't already have it, get it:
 
 ```
-go get github.com/magefile/mage
+go get github.com/magefile/mage@latest
```

Comment:
```suggestion
go install github.com/magefile/mage@latest
```

Reply:
@jmooring, okay, so what is the difference here and why does `go get` works for me in this case?

Reply:
```bash
go version
```

> go version go1.17.4 linux/amd64

```bash
rm $(which mage)
go get github.com/magefile/mage@latest
```

> go get: installing executables with 'go get' in module mode is deprecated.
	To adjust and download dependencies of the current module, use 'go get -d'.
	To install using requirements of the current module, use 'go install'.
	To install ignoring the current module, use 'go install' with a version,
	like 'go install example.com/cmd@latest'.
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.

```bash
which mage
```

> /home/jmooring/go/bin/mage